app.rb

```diff
@@ -0,0 +1,60 @@
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @peoples = []
+  end
```

Comment:
- [ ] Hey @Koshao-sys, You have done a good job so far but your app isn't working as expected. I would suggest that you initialize the @rentals instance variable. Check out this lesson to help you better understand this https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point)